cypress/config/development.json

```diff
@@ -0,0 +1,4 @@
+{
+    "video": true,
```

Comment:
@oluoluoxenfree I think we need to disable video on dev runs, but have a new revew.json file, with video enabled